pydicom/_storage_sopclass_uids.py

```diff
@@ -2,349 +2,4 @@
-Comprehensive3DSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.34')
-ExtensibleSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.35')
-ProcedureLogStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.40')
-MammographyCADSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.50')
-KeyObjectSelectionDocumentStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.59')
-ChestCADSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.65')
-XRayRadiationDoseSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.67')
-RadiopharmaceuticalRadiationDoseSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.68')
-ColonCADSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.69')
-ImplantationPlanSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.70')
-AcquisitionContextSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.71')
-SimplifiedAdultEchoSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.72')
-PatientRadiationDoseSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.73')
-PlannedImagingAgentAdministrationSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.74')
-PerformedImagingAgentAdministrationSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.75')
-EnhancedXRayRadiationDoseSRStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.88.76')
-TwelveLeadECGWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.1.1')
-GeneralECGWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.1.2')
-AmbulatoryECGWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.1.3')
-HemodynamicWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.2.1')
-CardiacElectrophysiologyWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.3.1')
-BasicVoiceAudioWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.4.1')
-GeneralAudioWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.4.2')
-ArterialPulseWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.5.1')
-RespiratoryWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.6.1')
-MultichannelRespiratoryWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.6.2')
-RoutineScalpElectroencephalogramWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.7.1')
-ElectromyogramWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.7.2')
-ElectrooculogramWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.7.3')
-SleepElectroencephalogramWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.7.4')
-BodyPositionWaveformStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.9.8.1')
-ContentAssessmentResultsStorage = UID(
-    '1.2.840.10008.5.1.4.1.1.90.1')
-RTBrachyApplicationSetupDeliveryInstructionStorage = UID(
-    '1.2.840.10008.5.1.4.34.10')
-RTBeamsDeliveryInstructionStorage = UID(
-    '1.2.840.10008.5.1.4.34.7')
-HangingProtocolStorage = UID(
-    '1.2.840.10008.5.1.4.38.1')
-ColorPaletteStorage = UID(
-    '1.2.840.10008.5.1.4.39.1')
-GenericImplantTemplateStorage = UID(
-    '1.2.840.10008.5.1.4.43.1')
-ImplantAssemblyTemplateStorage = UID(
-    '1.2.840.10008.5.1.4.44.1')
-ImplantTemplateGroupStorage = UID(
-    '1.2.840.10008.5.1.4.45.1')
+from pydicom.uid import *
```

Comment:
Should remove or change the file docstring since no longer auto-generating here. Aksi would be helpful to leave a comment that this import * is kept for backwards compatibility.